### PR TITLE
[12.0] [MIG] stock_picking_line_sequence

### PR DESCRIPTION
**Pull request by bjeficent**
Migration of module stock_picking_line_sequence to v12

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Review by AaronHForgeFlow — COMMENTED** [review]
I could not see the widget to move the lines in the pickings on runbot.

File: stock_picking_line_sequence/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    # displays sequence on the stock moves
+    sequence2 = fields.Integer('Sequence',
+                               help="Shows the sequence in the Stock Move.",
+                               related='sequence', readonly=True, store=True)
+
+    @api.model
+    def create(self, values):
+        move = super(StockMove, self).create(values)
+        # We do not reset the sequence if we are copying a complete picking
+        if self.env.context.get('keep_line_sequence'):
```
Comment (AdriaGForgeFlow) [inline stock_picking_line_sequence/models/stock.py]: Here you are doing the complete opposite thing of the comment above. If keep_line_sequence is true you are resetting the sequence.

File: stock_picking_line_sequence/models/stock.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        for rec in self:
+            current_sequence = 1
+            for line in rec.move_ids_without_package:
+                line.sequence = current_sequence
+                current_sequence += 1
+
+    @api.multi
+    def write(self, values):
+        res = super(StockPicking, self).write(values)
+        self._reset_sequence()
```
Comment (AdriaGForgeFlow) [inline stock_picking_line_sequence/models/stock.py]: Why is this here? This reset prevents the sequence to be kept if you are generating the picking from a sales order, for example.

**Review by AdriaGForgeFlow — COMMENTED** [review]
Some comments

**Comment by rousseldenis** [comment]
@bjeficent Do you plan to finish this ?

**Comment by AaronHForgeFlow** [comment]
@rousseldenis Yes, we want to fix this. Comments here by @ageficent were resolved in v10 and must be included here. Thanks

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: LGTM

**Comment by nikul-serpentcs** [comment]
@bjeficent @aheficent  Fix Transifex Commits `OCA Transbot updated translations from Transifex  …`

**Review by AdriaGForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by rousseldenis** [comment]
@bjeficent Could you squash your two commits ?

**Comment by AaronHForgeFlow** [comment]
419ed6a592be7d47c58d27a79b9cf2f1f4b64a23 comes before 417a4d5d53295bd383ad35a01f40ecbed9622138 but in the Pull Request appears to be later.

**Comment by pedrobaeza** [comment]
@aheficent I think this is GitHub reordering commits by date

**Comment by rousseldenis** [comment]
So, merging as is
